sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_encoder_protocols.py

```diff
@@ -8,6 +8,19 @@
 
 
 class SchemaContentValidate(Protocol):
+    # TODO: make __call__ a public API so that docs show, until then, keep below docstring
+
+    """
+    Callable protocol which validates content against provided schema. If invalid, raises Exception.
+     Else, returns None.
+
+    :param mapping[str, any] schema: The schema to validate against.
```

Comment:
you might want to double-check that this renders okay in the docs. I think the space in mapping[str, any] could be problematic

Reply:
It seems to be rendering _okay_? The `any` isn't linking to anything, but that's strange b/c the same format `dict[str, any]` is rendering fine in some places but not in others. is there anything specific about mapping that would render differently from dict?

Reply:
not so much the mapping, but I've seen the space between `str, any` cause it to completely break the type. If it looks good then ignore me. Maybe it was a :keyword: thing 🤷‍♀️